Apps/WebClient/src/ClientApp/src/components/navmenu/HeaderComponent.vue

```diff
@@ -24,285 +23,266 @@ import { ILogger } from "@/services/interfaces";
 
 library.add(faBars, faSignInAlt, faSignOutAlt, faTimes, faUser, faLightbulb);
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-const options: any = {
-    components: {
-        RatingComponent,
-        AppTourComponent,
-    },
-};
+const sidebarId = "notification-centre-sidebar";
+const minimumScrollChange = 2;
 
-@Component(options)
-export default class HeaderComponent extends Vue {
-    @Action("toggleSidebar", { namespace: "navbar" })
-    toggleSidebar!: () => void;
+const store = useStore();
+const route = useRoute();
+const router = useRouter();
+const logger: ILogger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
```

Comment:
```suggestion
const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
```
Type specification is unnecessary when using type argument with `container.get()`.